src/lib/pubkey/ec_group/ec_group.cpp

```diff
@@ -223,7 +223,7 @@ class EC_Group_Data_Map final
                {
                /*
                * If the same curve was previously created without an OID
-               * but has been registered again using an OID, save that OID.
+               * but is now being registered again using an OID, save that OID.
                */
                if(oid.empty() == false)
                   {
```

Comment:
Off-topic, but I believe the `throw Invalid_Argument("Cannot register ECC group ...` below is **unreachable** due to the `continue` above.

Reply:
This logic was pretty contorted overall. I rewrote the whole sequence to avoid any nested conditionals.

Reply:
OK, I think it's better now.